.env

```diff
@@ -0,0 +1,3 @@
+BASE_URL = https://www.saucedemo.com/
+USER_NAME = standard_user
```

Comment:
looks like this file wasn't used
To use it you need to read it and create env variables for the process of your tests
To do that you can use [dotenv](https://www.npmjs.com/package/dotenv) npm library